lib/jsdoc/util/logger.js

```diff
@@ -130,34 +141,26 @@ function addPrefix(args, prefix) {
     return updatedArgs || args;
 }
 
-// TODO: document events
-function wrapLogFunction(name, func) {
-    var eventName = 'logger:' + name;
-    var upperCaseName = name.toUpperCase();
-    var level = LEVELS[upperCaseName];
-    var prefix = PREFIXES[upperCaseName];
-
-    return function() {
-        var loggerArgs;
 
-        var args = Array.prototype.slice.call(arguments, 0);
-
-        if (logLevel >= level) {
-            loggerArgs = addPrefix(args, prefix);
-            func.apply(null, loggerArgs);
-        }
+var Logger = function () {
```

Comment:
Nit: For consistency with the rest of the file (and codebase), please omit the space after `function`. (Please fix the other occurrences as well, which I didn't call out.)